CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 ### Changes
 
+* Remove `stripPlaceholderBrs` and `restorePlaceholderBrs` from `AposRichTextWidgetEditor.vue` component.
```

Comment:
Should mention the addition of alt attribute support for inline images.

Reply:
Yes